lib/embed.js

```diff
@@ -1,19 +1,42 @@
 import renderTextItems from './text-items';
+const twitterRegex = /^https?:\/\/twitter\.com\/([-a-zA-Z0-9+&@#%?=~_|!:,.;]+)\/status(es){0,1}\/(\d+)/;
+
+function formatTwitterUrl (url) {
+  const match = url.match(twitterRegex);
+  if (!match) {
+    return '';
+  }
+
+  return `https://twitter.com/${match[1]}/status/${match[3]}`;
+}
```

Comment:
@kesla not sure, apple news requires twitter urls to match this regex, I'm removing embeds that do not match here. Do we want to add this requirement on article-json level?